sigstore/_internal/sct.py

```diff
@@ -48,18 +47,18 @@ def _pack_signed_entry(
         pack_format = "!BBB{cert_der_len}s"
         cert_der = cert.public_bytes(encoding=serialization.Encoding.DER)
     elif sct.entry_type == LogEntryType.PRE_CERTIFICATE:
-        if not issuer_key_hash or len(issuer_key_hash) != 32:
+        if not issuer_key_id or len(issuer_key_id) != 32:
             raise InvalidSctError("API misuse: issuer key hash missing")
```

Comment:
nit: comment update to `issuer key ID`

Reply:
Meaning you Want me to add an inline source comment here? I made this change primarily to make the naming consistent with RFC 6962, which refers to this identifier as a "key ID" rather than an issuer hash.

I'm happy to say as much in a source comment, as well.

Reply:
I just meant in the comment "API misuse: issuer key hash missing"

Reply:
Gotcha, thanks for catching that!